openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -101,17 +101,14 @@
     
     // membervariables
     /**
-     * This constant may only be changed by Paulo Soares and/or Bruno Lowagie.
      * @since	2.1.6
      */
```

Comment:
Would it also make sense to rename the variable?  If we're trying to remove anything to do with iText, which I'm fine with, then the variable name doesn't really make a lot of sense here.